openerp/addons/base/migrations/7.0.1.3/post-migration.py

```diff
@@ -60,72 +60,43 @@ def migrate_company(cr):
 
 
 def migrate_base_contact(cr):
-    """
-    Move entries of res_partner_contact into res_partner
-    """
+    """Move data of res_partner_contact into res_partner."""
     cr.execute(
         "SELECT * FROM ir_module_module "
         "WHERE name = 'base_contact' and state = 'to remove';")
     if not cr.fetchall():
         return
-    fields = [
-        'create_date',
-        'name',
-        'website',
-        'image',
-        'active',
-        'comment',
-        'title',
-        'phone',
-        'country_id',
-        'email',
-        'birthdate',
-        'lang',
-        'parent_id',
-    ]
-    # Add lang from lang_id
+    # Add non-existing columns that can be used with partner_lastname module
+    cr.execute("ALTER TABLE res_partner "
+               "ADD COLUMN firstname character varying;")
```

Comment:
I don't think they're going to exist, because all the possible references to last and first name are on contact or res.partner.address.